ScanCommand/src/test/java/io/ballerina/scan/Main.java

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
Just a suggestion: It is better file name reflect the actual usecase. 
TestScanTool.java or something like that.
wdyt?

Reply:
Fixed in [5c9e9f8](https://github.com/ballerina-platform/static-code-analysis-tool/pull/2/commits/5c9e9f806ffb2fcb8658f63f7e99981c74ceed04)